WordPress/src/main/res/layout/reader_interest_card.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/interest_container"
+    style="@style/ReaderCardView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/root_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/interests_list"
+            android:layout_marginTop="@dimen/margin_extra_large"
+            android:layout_width="match_parent"
```

Comment:
Np: We should try to use `0dp` instead of `match_parent` for widgets contained in a `ConstraintLayout`. From [official doc](https://developer.android.com/reference/android/support/constraint/ConstraintLayout.html#RelativePositioning)

> Important: MATCH_PARENT is not recommended for widgets contained in a ConstraintLayout.  Similar behavior can be defined by using MATCH_CONSTRAINT with the corresponding left/right or top/bottom constraints being set to "parent".

Reply:
Oh yes, thanks for this!

Reply:
Resolved here 0d668b4